pkg/server/serverpb/status.proto

```diff
@@ -1484,9 +1484,9 @@ message HotRangesResponseV2 {
     // table_name has been deprecated in favor of tables = 16;
     string table_name = 4 [deprecated = true];
     // database_name has been deprecated in favor of databases = 17;
-    string database_name = 5;
+    string database_name = 5 [deprecated = true];
```

Comment:
I think typically we opt for `reserved` - is there a reason we're doing a softer deprecation here?

Reply:
No reason other than muscle memory - I'll change this.